src/config.ts

```diff
@@ -10,8 +10,14 @@ const defaults = {
     DB_CONNECTION_STRING: "mongodb://localhost/fsd1?authSource=admin",
     ENVIRONMENT: Environment.DEV,
     PORT: 3000,
+    REFRESH_TOKEN_EXPIRES: "7d",
+    TOKEN_EXPIRES: "1h",
+    TOKEN_SECRET: "6e44b1fb1a26aa1c7324b6f7b555de5446c0fa22ef02b3f176523a56c85094f7a5912ddad3b70e59f70da2ca7e75566a9c52d41adf8ae82cf9ed27106dffa103",
```

Comment:
Since the default value is for dev, it is better to give it a value like `changme`
```suggestion
    TOKEN_SECRET: "changeme",
```